app/services/facebook_authenticator.rb

```diff
@@ -23,15 +27,15 @@ def authenticate!
 
   private
 
-    def login
+    def find_login(facebook_user)
       @login ||= Login.where(identification: facebook_user[:email]).first
```

Comment:
let's only cache that if we find it's necessary. The method should only ever be invoked once anyway.